<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayer</class>
 <widget class="QWidget" name="VideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0,0,0);
border: none;
color: rgb(255,255,255);
QPushButton::menu-indicator{ 
    image: url(none); 
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGst::Ui::VideoWidget" name="video" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="bufferProgress">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="overlay" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>38</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">    border: 1px dashed rgb(255,255,255);
    border-radius: 4px;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::HLine</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame">
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="lineWidth">
               <number>4</number>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="playPauseButton">
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>202</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>P</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="timeSlider">
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QToolButton" name="volume">
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>V</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="settings">
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>S</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGst::Ui::VideoWidget</class>
   <extends>QWidget</extends>
   <header>QGst/Ui/videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
